docs/vt-extensions/color-palette-update-notifications.md

```diff
@@ -64,3 +64,4 @@ in order to see how to implement this in your own application.
 | not yet  | Helix                | see tracker: [helix#13281](https://github.com/helix-editor/helix/issues/13281)                      |
 | not yet  | WezTerm              | see tracker: [wezterm#6454](https://github.com/wez/wezterm/issues/6454)                             |
 | not yet  | Zellij               | see tracker: [zellij#3831](https://github.com/zellij-org/zellij/issues/3831)                        |
+| not yet  | VTE                  | see tracker: [vte#2880](https://gitlab.gnome.org/GNOME/vte/-/issues/2880)                           |
```

Comment:
I suspect that issue will end up being closed as a duplicate of [vte#2740](https://gitlab.gnome.org/GNOME/vte/-/issues/2740).

Reply:
We would change the link then :)

Reply:
Ah very nice. Makny thanks to all of you.